### DISCUSSION TITLE
KDE Plasma Version - recent applications issue

### DISCUSSION THREAD

**Discussion by bin-linux** [SpiralLinux/SpiralLinux-project#9] [General]
Hello - I'm testing Spiral KDE Plasma in VM.
Using the Application Menu (cascading popup) I have noticed that the Recent Applications does not record some apps.
So far these are affected:-
Dolphin
Synaptic
Konsole
Kate
System Settings
There may be others but not found them yet.

**Comment by geckolinux** [comment]
Hi there, would it maybe be due to having opened those applications from the favorites and/or panel shortcuts? As far as I know the menu only records recent applications that were opened by going into one of the main menu categories.

**Reply by bin-linux** [reply]
Thanks - that was my first thought but it also applies when opening from the main menu. I've tried menu resets and a few other things but no result. It is a bit odd, but having tried a couple of other Debian Plasma distros in the mean time it seems a common thing. Hmmm puzzled - but no matter. I've been looking for a config file for Recent Applications but doesn't seem to be one....

**Reply by geckolinux** [reply]
Ok, thanks for confirming that. I guess it's an upstream Plasma thing, or at least in the version that Debian ships.

**Comment by bin-linux** [comment]
Just out of interest and to close the circle - updated to Plasma 5.24 and still has the same issue.

**Reply by geckolinux** [reply]
Thanks for confirming that! Was that on a different distro, or did you upgrade SpiralLinux to Testing/Sid?

**Comment by bin-linux** [comment]
It was done on Spiral using Norbert Preining's Debian updates https://www.preining.info/blog/2022/05/kde-gears-22-04-and-plasma-5-24-5-for-debian/

**Reply by geckolinux** [reply]
Ah OK, so an add-on repo on top of SpiralLinux with the latest Plasma/KDE? In that case I still suspect it's a KDE bug, but there still exists a possibility that it's a SpiralLinux configuration issue or a missing package or something along those lines. Maybe you could try the Debian Stable live KDE spin to see if it's the same. Thanks!

**Comment by bin-linux** [comment]
OK - I've tried various Plasma based Debian versions and the same issue appears across the range in one form or another. Oddly in a couple of cases Konsole does work as expected from Favourite>Recent Applications. I've not found anything significant in KDE Bugzilla. It is certainly nothing specific to Spiral that's for sure. :)

**Reply by geckolinux** [reply]
Thanks a lot, really appreciate you confirming that.
